catroid/src/androidTest/java/org/catrobat/catroid/common/firmata/FirmataUtils.java

```diff
@@ -44,7 +51,24 @@ public FirmataUtils(ConnectionDataLogger logger) {
 		this.logger = logger;
 	}
 
-	public FirmataMessage getAnalogMesageData() {
+	public FirmataMessage getDigitalMessageData() {
+
+		int portAndCommand = getNextMessage();
+
+		int port = getPinFromHeader(portAndCommand);
+		int command = getCommandFromHeader(portAndCommand);
+
+		Assert.assertEquals("This is no digital message command", DIGITAL_MESSAGE_COMMAND, command);
```

Comment:
This method is supposed to be used in tests only. The original methods in this file use a similar style. Also [BluetoothTestUtils.java](https://github.com/Catrobat/Catroid/blob/develop/catroid/src/androidTest/java/org/catrobat/catroid/common/bluetooth/BluetoothTestUtils.java) uses a similar style. If you still want me to use an exception here, could you suggest which exception to use?